test/files/neg/implicitly-self.check

```diff
@@ -0,0 +1,12 @@
+implicitly-self.scala:3: warning: Implicit resolves to enclosing symbol method c
+  implicit def c: Char = implicitly[Char]
+                                   ^
+implicitly-self.scala:4: warning: Implicit resolves to enclosing symbol value s
+  implicit val s: String = implicitly[String]
+                                     ^
+implicitly-self.scala:6: warning: Implicit resolves to enclosing symbol value t
```

Comment:
I would bikeshed for `resolves to enclosing value t`, &c.

Reply:
Yep.